components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -13,55 +13,52 @@
 #include "base/containers/flat_map.h"
 #include "base/files/file_path.h"
 #include "base/files/file_util.h"
-#include "base/path_service.h"
-#include "chrome/common/chrome_paths.h"
 #include "base/files/important_file_writer.h"
 #include "base/guid.h"
+#include "base/i18n/time_formatting.h"
```

Comment:
nit: to reduce the size of this PR can we only sort newly added `#include`'s for legacy .h/.cc files